src/cosmology/api/standard.py

```diff
@@ -53,7 +62,17 @@
 
 
 @runtime_checkable
-class StandardCosmologyAPI(BackgroundCosmologyAPI[ArrayT], Protocol):
+class StandardCosmologyAPI(
+    NeutrinoComponent[ArrayT],
+    BaryonComponent[ArrayT],
+    PhotonComponent[ArrayT],
+    DarkMatterComponent[ArrayT],
+    MatterComponent[ArrayT],
+    DarkEnergyComponent[ArrayT],
+    GlobalCurvatureComponent[ArrayT],
+    BackgroundCosmologyAPI[ArrayT],
+    Protocol,
+):
```

Comment:
Hmmm, if I look at CAMB, CLASS, and Astropy, the component interface you made is so simple that we could in principle get rid of the StandardCosmology altogether.

When defining each wrapper, you can pick exactly which components you support. That seems much more robust now, because as soon as a wrapped cosmology does not perfectly agree with the standard choice made here, you need to do it anyway.

Reply:
It is very convenient! But I think it's nice to have a single "standard" cosmology that is some form of ΛCDM.